ledger/ledgercore/statedelta.go

```diff
@@ -563,10 +496,10 @@ func (ad NewAccountDeltas) GetBasicsAccountData(addr basics.Address) (basics.Acc
 
 // ToModifiedCreatables creates map of ModifiedCreatable
 func (ad NewAccountDeltas) ToModifiedCreatables(seen map[basics.CreatableIndex]struct{}) map[basics.CreatableIndex]ModifiedCreatable {
-	result := make(map[basics.CreatableIndex]ModifiedCreatable, len(ad.appParams)+len(ad.assetParams))
-	for aapp, idx := range ad.appParamsCache {
-		rec := ad.appParams[idx]
-		if rec.Params == nil {
+	result := make(map[basics.CreatableIndex]ModifiedCreatable, len(ad.appResources)+len(ad.assetResources))
```

Comment:
please add a comment on this method that it's only being used in tests.

Reply:
done! additionally, if we wanted to get really paranoid, we could put a init() function is _test.go code to set an env variable or flag that needs to be true to use functions like these.